yoti-sdk-spring-boot-example/src/main/resources/templates/profile.html

```diff
@@ -1,88 +1,99 @@
+
+            <section class="yoti-attributes-section">
+
+                <div th:if="${appLogo == null}">
+                    <img class="yoti-company-logo" th:src="@{/static/assets/company-logo.jpg}" alt="company logo" />
+                </div>
+                <div th:if="${appLogo != null}">
+                    <img class="yoti-company-logo" th:src="${appLogo}" alt="company logo" />
+                </div>
+
+                <div class="yoti-attribute-list-header">
+                    <div class="yoti-attribute-list-header-attribute">Attribute</div>
+                    <div class="yoti-attribute-list-header-value">Value</div>
+                    <div>Anchors</div>
+                </div>
+
+                <div class="yoti-attribute-list-subheader">
+                    <div class="yoti-attribute-list-subhead-layout">
+                        <div>S / V</div>
+                        <div>Value</div>
+                        <div>Sub type</div>
+                    </div>
+                </div>
+
+                <div class="yoti-attribute-list">
+
+                    <div th:each="attribute : ${displayAttributes}" class="yoti-attribute-list-item">
+
+                        <div class="yoti-attribute-name">
+                            <div class="yoti-attribute-name-cell">
+                                <i th:class="${attribute.icon}" />
+                                <span class="yoti-attribute-name-cell-text" th:text="${attribute.displayName}" />
+                            </div>
+                        </div>
+
+                        <div class="yoti-attribute-value">
+                            <div class="yoti-attribute-value-text" th:text="${attribute.displayValue}" />
+                        </div>
+
+                        <div class="yoti-attribute-anchors-layout">
+                            <div class="yoti-attribute-anchors-head -s-v">S / V</div>
+                            <div class="yoti-attribute-anchors-head -value">Value</div>
+                            <div class="yoti-attribute-anchors-head -subtype">Sub type</div>
+
+                            <div th:each="anchor : ${attribute.attribute.sources}">
+                                <div class="yoti-attribute-anchors -s-v">Source</div>
+                                <div class="yoti-attribute-anchors -value" th:text="${anchor.value}" />
+                                <div class="yoti-attribute-anchors -subtype" th:text="${anchor.subType}" />
+                            </div>
+
+                            <div th:each="anchor : ${attribute.attribute.verifiers}">
+                                <div class="yoti-attribute-anchors -s-v">Verifier</div>
+                                <div class="yoti-attribute-anchors -value" th:text="${anchor.value}" />
+                                <div class="yoti-attribute-anchors -subtype" th:text="${anchor.subType}" />
+                            </div>
+                        </div>
+
```

Comment:
Did you add these blank lines manually or is it your linter?

Reply:
I would have done that

Reply:
I know you're matching how the tags were opened, but I think it's tidier without line breaks, especially when you don't have line breaks after some of the div tags. But no problem if you want to keep it like this!

Reply:
I prefer the line breaks, keeping divs in clear blocks.  I'm going to leave it...